packages/backend/src/api/v1/worksheet-filter-pivot.controller.ts

```
@@ -0,0 +1,145 @@
+
+  @Get('filter')
+  async getFilter(
+    @Param('workspaceId') workspaceId: string,
+    @Param('documentId') documentId: string,
+    @Param('tabId') tabId: string,
+  ) {
+    await this.assertSheetDocument(documentId, workspaceId);
+    return this.yorkieService.withDocument(
+      documentId,
+      (doc) => {
+        const ws = doc.getRoot().sheets?.[tabId] as
```

ESLint

[error] Unsafe call of a type that could not be resolved.

(`@typescript-eslint/no-unsafe-call`)

---

[error] Unsafe member access .getRoot on a type that cannot be resolved.

(`@typescript-eslint/no-unsafe-member-access`)

---

[error] Unsafe member access .sheets on a type that cannot be resolved.

(`@typescript-eslint/no-unsafe-member-access`)

---

[error] Replace `|·{·filter?:·unknown·}⏎·········` with `{·filter?:·unknown·}`

(prettier/prettier)